index.html

```diff
@@ -1377,6 +1397,10 @@ <h4>Additional Constraints</h4>
             applied to the Interaction Affordances of a <a>Thing
               Description</a> conforming to the <a>Core
               Data Model</a>:
+	     <em>
+	     FIX: this should have an table assertion per row, and this should be informative
+	     or a parent assertion.
+	     </em>
```

Comment:
This one is annoying because unlike other tables, it's not saying these elements are mandatory, it's actually a list of assertions.  Converting into a list of assertions, therefore.  The "_" is used to indicate these are Child assertions of the main assertion to keep the structure (although we could also just make the introductory paragraph informative).  Which DID require renumbering them since these were all 1, 2, 3 etc even though 2 was formerly a sub-part of 1, etc.

Reply:
I must say several of these are assertions which should have been added to TDs as bug fixes.

Reply:
to avoid getting into arguments about formatting, I left things as a table but use regular (child) assertions inside the table.  I did reword things a bit to make the assertions more self-contained and avoid starting sentences with lower case letters.